<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LongArifmetic</class>
 <widget class="QMainWindow" name="LongArifmetic">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LongArifmetic</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="firstNumber">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>30</y>
      <width>681</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="secondNumber">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>681</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="result">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>270</y>
      <width>681</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="choise_add">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Прибавить</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choise_sub">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>80</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Вычесть</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choise_mul">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Умножить</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choise_div">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Поделить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>210</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Посчитать</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choise_fact">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>82</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Факториал</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="choise_pow">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Возвести в степень</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
